python/snewpy/snowglobes_interface.py

```diff
@@ -218,8 +218,6 @@ class Runner:
     def __post_init__(self):
         self.channels=self.sng.channels[self.material]
         self.binning=self.sng.binning[self.material]
-        self.efficiency=self.sng.efficiencies[self.detector]
```

Comment:
Is there a reason for removing `self.efficiency` and `self.smearing` initialization? 
Probably they need to be restored, for the Runner class to work

Reply:
This was an accident, I'll add it back